src/thunderbots/software/CMakeLists.txt

```diff
@@ -41,6 +41,12 @@ file(GLOB SHARED_UTIL_SRC LIST_DIRECTORIES false CONFIGURE_DEPENDS
         ${CMAKE_CURRENT_SOURCE_DIR}/../shared/*.h
         )
 
+# Find the g3log library, the logger we use for our AI
+# See: https://github.com/KjellKod/g3log
+SET(G3_LIBRARY_PATH "/usr/local/include")
+message("g3log lib path: ${G3_LIBRARY_PATH}")
```

Comment:
These messages seem like they're there for debugging; consider removing?

Reply:
I'm thinking of leaving them in because they are similar to the protobuf messages like `Found PROTOBUF Compiler: /usr/bin/protoc` that I find useful as status reports as CMake runs. Do you have a strong preference?